### DISCUSSION TITLE
What is the relationship between Solana and Velas?

### DISCUSSION THREAD

**Discussion by for-james** [velas/velas-chain#272] [Q&A]
Why does [the example](https://docs.velas.com/apps/webwallet) use Solana's repository? What is the difference with Solana and Velas ?

**Comment by vldm** [comment]
Velas codebase is forked from solana (with solana agreement), but velas has other ideas and plans.
For example in velas we also support evm txs, and evm address space, which makes us more friendly to ethereum eco-system.
